mods/ctf/ctf_modebase/bounties.lua

```diff
@@ -137,17 +143,56 @@ ctf_core.register_chatcommand_alias("list_bounties", "lb", {
 	func = function(name)
 		local pteam = ctf_teams.get(name)
 		local output = {}
+		local x = 0
+		local y = 0
 
 		for tname, bounty in pairs(bounties) do
 			if pteam ~= tname then
-				table.insert(output, bounty.msg)
+				local label = string.format("label[%d,%d;%s]", x, y, bounty.name .. " => " .. bounty.rewards.score)
+				y = y + 1
+				table.insert(output, label)
+				local model = "model[%d,%d;4,6;player;character.b3d;%s;{0,160};;;]"
+				model = string.format(
+					model,
+					x,
+					y,
+					ctf_cosmetics.get_colored_skin(bounty.name, ctf_teams.get(bounty.name))
+				)
+				table.insert(output, model)
+				if y > 8 then
+					y = 0
+				else
+					x = x + 4.5
+				end
 			end
 		end
 
 		if #output <= 0 then
 			return false, "There are no bounties you can claim"
 		end
-
-		return true, table.concat(output, "\n")
+		
+		local formspec = "size[10,10]\n" .. table.concat(output, "\n")
+		minetest.show_formspec(name, "ctf_modebase:lb", formspec)
+		return true, ""
 	end
 })
+
+ctf_core.register_chatcommand_alias("put_bounty", "pb", {
+	description = "Put bounty on some player",
+	params = "<player> <amount>",
+	privs = {ctf_admin=true},
+	func = function(name, param)
+		local player, amount = string.match(param, "(.*) (.*)")
+		local pteam = ctf_teams.get(player)
+		if not (player and pteam and amount) then
+			return false, "Incorrect parameters"
+		end
+		amount = ctf_core.tonumber(amount)
+		set(
+			player,
+			pteam,
+			{ bounty_kills=1, score=amount }
+		)
+		return true, "I just put " .. amount .. " on " .. player
```

Comment:
```suggestion
		return true, "Successfully placed a bounty of " .. amount .. " on " .. player .. "!"
```

Since many chat-commands and other server-sent chat messages now colorize the player name according to their team, I'd suggest colorizing the player's name here as well, for consistency.

`minetest.colorize("#rrggbb", player)`